src/components/Row/coinRow.js

```diff
@@ -138,5 +141,49 @@ export function CoinRow({ activeCoin, coinList, updateSelectedCoins }) {
 		);
 	});
 
-	return <ScrollRow>{coinButtons}</ScrollRow>;
+	const [maxScrollLeft, setMaxScrollLeft] = useState(0);
+
+	useEffect(() => {
+		setMaxScrollLeft(scroll.current.scrollWidth - scroll.current.clientWidth);
+	}, []);
+
+	function leftScroll() {
+		// take 4 clicks to get to the end of a scroll
+		scroll.current.scrollLeft -= maxScrollLeft / 3;
+	}
+
+	function rightScroll() {
+		scroll.current.scrollLeft += maxScrollLeft / 3;
+	}
+
+	const [showLeftArrow, setShowLeftArrow] = useState(false);
+	const [showRightArrow, setShowRightArrow] = useState(true);
```

Comment:
Move these up top with other state, nice to keep these all at the top of the function so we can quickly know what state the function holds.